packages/frontend/tailwind.config.js

```diff
@@ -7,6 +7,9 @@ export default {
       "tennis-green": "#2ecc71",
       "squash-yellow": "#f1c40f",
       "font-gray": "#1c3e50",
+      "font-white": "#ffffff",
```

Comment:
### Naming
Notice that Tailwind's colours are generally not prefixed (unless it's a shade of a colour).

 ```suggestion
      white: "#ffffff",
```